op-e2e/op_geth.go

```diff
@@ -90,15 +120,21 @@ func NewOpGeth(t *testing.T, ctx context.Context, cfg *SystemConfig) (*OpGeth, e
 	)
 	require.Nil(t, err)
 
-	l2Client, err := ethclient.Dial(node.HTTPEndpoint())
+	l2Client, err := ethclient.Dial(HTTPEndpoint)
+	require.Nil(t, err)
+
+	l2RpcClient, err := rpc.Dial(HTTPEndpoint)
 	require.Nil(t, err)
 
 	genesisPayload, err := eth.BlockAsPayload(l2GenesisBlock)
 
 	require.Nil(t, err)
 	return &OpGeth{
 		node:          node,
+		opNode:        opNode,
```

Comment:
Any reason to store this in the structure?  I'd suggest that we store simply an `EthInstance` instead of the node/opNode.  This is a little more symmetric with how it's done in the more full end to end, and it will force tests to manually cast the eth instance to an op node if they want to directly mess with it.